src/clean-aws-credentials-cache/index.js

```diff
@@ -0,0 +1,15 @@
+const fs = require('fs');
+
+//https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_revoke-sessions.html
+function cleanAwsCredentialsCache() {
```

Comment:
Could you introduce some sanity checking by platform? These should never both execute, but technically it is possible on a *nix system to have a directory named `%UserProfile%`.

Reply:
I've added a `proccess.platform` validation to ensure we are only attempting to cleanup window files on a windows system.

Addressed in ca01d49

Reply:
Second commit to get things working:
ef119d1